CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to this project will be documented in this file. If a contri
 - Refactor out theme logic in StyledComponent's componentWillMount & componentWillReceiveProps (see [#1130](https://github.com/styled-components/styled-components/issues/1130))
 - Add onReset to valid react props list (see [#1234](https://github.com/styled-components/styled-components/pull/1234))
 - Add support for ServerStyleSheet PropType in both StyleSheetManager and StyledComponent (see [#1245](https://github.com/styled-components/styled-components/pull/1245))
+- Added basic support for style objects in the Typescript definitions (see [#1123](https://github.com/styled-components/styled-components/pull/1123))
```

Comment:
Just very minor comment, that for consistency with other entries in the change log, use imperative tense here: _"Add~~ed~~ basic..."_